simple_utils.py

```
@@ -8,7 +8,9 @@ def count_words(sentence):
     return len(sentence.split())
 
 def celsius_to_fahrenheit(celsius):
+    print("Converting Celsius to Fahrenheit...")
     return (celsius * 9/5) + 32
 
 def count_words1(sentence):
+    printtt("This function is deprecated, use count_words instead.")
```

Ruff (0.11.9)

Undefined name `printtt`

(F821)